Library/Formula/python3.rb

```diff
@@ -142,6 +142,9 @@ def install
     ["Headers", "Python", "Resources"].each { |f| rm(prefix/"Frameworks/Python.framework/#{f}") }
     rm prefix/"Frameworks/Python.framework/Versions/Current"
 
+    # Symlink the pkgconfig files into HOMEBREW_PREFIX so they're accessible.
+    (lib+"pkgconfig").install_symlink Dir["#{frameworks}/Python.framework/Versions/3.4/lib/pkgconfig/*"]
```

Comment:
Fixed up ` + ` into `/`. Changed Python3's ` 3.4 ` into `#{xy}`. Didn't change ` Current ` on `python.rb`. Just poke if that needs changing to anything else as well.